packages/main/src/themes/sap_horizon/Panel-parameters.css

```diff
@@ -6,6 +6,7 @@
 	--_ui5_panel_background_color: var(--sapGroup_ContentBackground);
 	--_ui5_panel_border_radius: var(--sapElement_BorderCornerRadius);
 	--_ui5_panel_border_bottom: none;
-	--_ui5_panel_outline_offset: 0px;
+	--_ui5_panel_outline_offset: -0.0625rem;
```

Comment:
I am not sure, whether it will fit, but could you please check, how it will looks like with 0.125rem offset, because the focus outline is a little bit out of the panel now

Reply:
Done